src/core/common/permissions/types.ts

```diff
@@ -0,0 +1,120 @@
+import { isSiteModerationScoped } from "./isSiteModerationScoped";
```

Comment:
A lot of the types here are redundant but are needed (as far as I know) because we can't shouldn't import + depend on types from the backend in common. Please correct me if wrong!